### PR TITLE
B/serve admin ui from root

### PR DESCRIPTION
**Pull request by bogensberger**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,13 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Updated crate-admin to ``1.1.0`` which includes the following change:
+
+    - Breaking: Updated the admin-ui build to be compatible with future crate
```
Comment (mfussenegger) [inline CHANGES.txt]: this is not relevant for users - might even confuse them.

File: CHANGES.txt [hunk 2]
```diff
@@ -5,6 +5,13 @@ Changes for Crate
...
+      versions which will serve the admin-ui from root `/`.
+
+ - Serve admin ui from `/`
```
Comment (mfussenegger) [inline CHANGES.txt]: Isn't this already the case?

Comment (bogensberger) [inline CHANGES.txt]: currently it's served from '/admin'. '/' just redirects to '/admin'

Comment (mfussenegger) [inline CHANGES.txt]: Guess the user-facing breaking change is that the admin-ui can no longer be accessed via `/_plugin/crate-admin/#/`  but only `/` or `/admin`

**Pull request merged**